addons/web_editor/views/snippets.xml

```diff
@@ -384,6 +384,16 @@
         </we-row>
     </div>
 
+    <div data-js="LinkTool" data-selector="img, .media_iframe_video, span.fa, i.fa">
```

Comment:
So just patch the existing ReplaceMedia class in JS, don't create a new one :+1: